ckanclient/__init__.py

```diff
@@ -0,0 +1,165 @@
+import logging
+
+import requests
+from frictionless_ckan_mapper import ckan_to_frictionless, frictionless_to_ckan
+
+from ckanclient.auth import CkanAuthApi
+from ckanclient.text import camel_to_snake
+from ckanclient.upload import push_data_to_blob_storage, verify_upload
+
+
+LOG_FORMAT = '%(asctime)s - %(levelname)s - %(message)s'
+logging.basicConfig(format=LOG_FORMAT, level=logging.INFO)
+log = logging.getLogger(__name__)
+
+
+class CkanClientError(Exception):
+    pass
+
+
+class CkanClient:
```

Comment:
Just `Client` - we already know this is CKAN 😉

Reply:
No docstring. I recommend we put docs in docstring esp of parameters and then extract automatically to README.